src/node/cli.ts

```diff
@@ -542,6 +542,10 @@ export async function setDefaults(cliArgs: UserProvidedArgs, configArgs?: Config
     args.password = process.env.PASSWORD
   }
 
+  if (process.env.CS_DISABLE_FILE_DOWNLOADS) {
```

Comment:
✍️ This check only looks for the existence of the env var. As long as it is a truthy value (i.e. _not_ ""), it will set to `true`. Open to other approaches as well.

Reply:
Hmmm maybe we should do an explicit `true` and/or `1`, like `process.env.CS_DISABLE_FILE_DOWNLOADS === "true"`, just because it feels weird if you did `CS_DISABLE_FILE_DOWNLOADS=false code-server` and it still got enabled :stuck_out_tongue:

Reply:
Hahaha I knew there was a hole in this 🙈 Yeah, I like explicitness! I'll go with your suggestion